docs/en/reference/yaml-mapping.rst

```diff
@@ -1,6 +1,10 @@
 YAML Mapping
 ============
 
+.. attention::
```

Comment:
Do we have an `attention` directive? I don't see it in https://github.com/doctrine/rst-parser/tree/master/lib/Directives or https://github.com/doctrine/doctrine-website/tree/master/lib/RST/Directive

Reply:
I didn't know that not all rst-syntax is available with rst-parser. Better I'll generate my local files with rst-syntax before I introduce changes instead of using other rst-tools.

Reply:
I've changed it to "note". This one is available in doctrine/rst-parser.

Reply:
Can we have a `::big-blinking-red-alert` for this specific purpose? :smiley: